internal/view/log_indicator.go

```diff
@@ -47,6 +48,7 @@ func NewLogIndicator(cfg *config.Config, styles *config.Styles, allContainers bo
 func (l *LogIndicator) StylesChanged(styles *config.Styles) {
 	l.SetBackgroundColor(styles.K9s.Views.Log.Indicator.BgColor.Color())
 	l.SetTextColor(styles.K9s.Views.Log.Indicator.FgColor.Color())
+	l.Refresh()
```

Comment:
Why do we need `refresh` here?

Reply:
Hi @slimus , I added the refresh to have the colors automatically change when the skin.yml file is modified. Without this, the color's would not dynamically update after modifying skin.yml only after closing and reopening k9s.